tools/generate-helpers/bootstrap-migration/seq_num.go.tpl

```diff
@@ -0,0 +1,18 @@
+
+package internal
+
+var (
+	// CurrentDBVersionSeqNum is the current DB version number.
+	// This must be incremented every time we write a migration.
+	// It is a shared constant between central and the migrator binary.
+	CurrentDBVersionSeqNum = {{.nextSeqNum}}
+
+	// MinimumSupportedDBVersionSeqNum is the minimum DB version number
+	// that is supported by this database.  This is used in case of rollbacks in
+	// the event that a major change introduced an incompatible schema update we
+	// can inform that a rollback below this is not supported by the database
+	MinimumSupportedDBVersionSeqNum = {{.nextSeqNum}}
```

Comment:
Don't automatically update this one.  Should probably add it to one of the TODOs to evaluate if it needs to be updated.